grpc/oob/bmc/users_bmclibv2.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/bmc-toolbox/bmclib/v2"
 	"github.com/go-logr/logr"
 	v1 "github.com/tinkerbell/pbnj/api/v1"
+	common "github.com/tinkerbell/pbnj/grpc/oob"
```

Comment:
why is this being aliased to `common`? Is it just so that the uses below look similar to the ones above (grpc/oob/bmc/bmc.go for example)?

Reply:
Yep, that was the only reason to alias it to `common`